app/controllers/admin/cms/transcripts_controller.rb

```diff
@@ -0,0 +1,61 @@
+class Admin::Cms::TranscriptsController < Admin::ApplicationController
+  before_action :set_transcript, only: [:edit, :update]
+
+  def new
+    @transcript = Transcript.new(collection_id: collection_id)
```

Comment:
if we have the collection, then another way would be: `@collection.build_transcript`